mps-model-adapters/src/main/kotlin/org/modelix/model/mpsadapters/MPSLanguageRepository.kt

```diff
@@ -39,8 +38,6 @@ data class MPSLanguageRepository(private val repository: SRepository) : ILanguag
 
         val conceptDescriptor = ConceptRegistry.getInstance().getConceptDescriptor(conceptId)
 
-        if (conceptDescriptor is IllegalConceptDescriptor) return null
```

Comment:
Is this fixing the test added in 19cccb9f? (If yes, could we combine both commits to make it more clear?)

Reply:
No, this is WIP but might be a necessary step towards the solution.